.github/workflows/pre-commit.yml

```diff
@@ -13,3 +13,18 @@ jobs:
     - uses: pre-commit/action@v2.0.0
       env:
         SKIP: rstcheck
+
+    - name: "Generate patch file"
```

Comment:
```suggestion
    - name: "Generate pre-commit.patch"
```

Reply:
This is part of the pre commit workflow, so the original title is fine IMHO. I'd also prefer to keep the workflow consistent across all repos.

Reply:
Ok, It was just an idea for improvement.